.env.example

```diff
@@ -1,40 +1,46 @@
+# All possible environment variables for the frontend (set at build time **will end up in source code!!!**)
+
 # Network the CDapp is deployed to
-NETWORK=ganache
+VITE_NETWORK="ganache"
 
 # EtherRouter contract address for the network the CDapp is deployed to
 # DOES NOT NEED TO BE SET WHEN DEVELOPING LOCALLY
-NETWORK_CONTRACT_ADDRESS=0x0000000000000000000000000000000000000000
+VITE_NETWORK_CONTRACT_ADDRESS="0x0000000000000000000000000000000000000000"
 
-URL="http://localhost:9091"
+# Required for safe control
+VITE_BSCSCAN_API_KEY=
+VITE_ETHERSCAN_API_KEY=
 
-# URL where the authentication server is found
-AUTH_PROXY_ENDPOINT="http://localhost:3005"
+# The endpoint of the reputation oracle.
+VITE_REPUTATION_ORACLE_ENDPOINT="http://localhost:3001/reputation/local"
 
 # If the network has / or does not have support for metatranasctions
 # Ie: contracts deployed, broadcaster service active
-METATRANSACTIONS=true
-BROADCASTER_ENDPOINT="http://localhost:3004"
-
-# The endpoint of the reputation oracle.
-REPUTATION_ORACLE_ENDPOINT="http://localhost:3001/reputation/local"
+VITE_METATX_ENABLED=true
+VITE_METATX_BROADCASTER_ENDPOINT="http://localhost:3004"
 
-# The endpoint of the service where ganache-accounts.json can be found for frontend
-GANACHE_ACCOUNTS_ENDPOINT="http://localhost:3006"
+# The endpoint of the service where etherrouter-addres.json, ganache-accounts.json & safe-addresses.json can be found for frontend
+VITE_NETWORK_FILES_ENDPOINT="http://localhost:3006"
+VITE_GANACHE_RPC_URL="http://localhost:8545"
 
-# Required for safe control
-BSCSCAN_API_KEY=<bscscan-api-key>
-ETHERSCAN_API_KEY=<etherscan-api-key>
+# URL where the authentication server is found
+VITE_AUTH_PROXY_ENDPOINT="http://localhost:3005"
 
 # ID for Google Tag Manager `GTM-XXXXXXX`
-GOOGLE_TAG_MANAGER_ID=<gtm_id>
+VITE_GOOGLE_TAG_MANAGER_ID=
 
 # pinata connection detail from https://app.pinata.cloud/developers/api-keys
 # Only needed for production
-PINATA_API_KEY=<pinata-api-key>
-PINATA_API_SECRET=<pinata-api-secret>
-
-# Only needed to be changed for on-demand environment
-WEBPACK_DISABLE_HOT_RELOAD=false
+VITE_PINATA_API_KEY=
+VITE_PINATA_API_SECRET=
 
 # Needed for currency conversion. For testing, see: https://support.coingecko.com/hc/en-us/articles/21880397454233
-COINGECKO_API_KEY=<coingecko-api-key>
+VITE_COINGECKO_API_KEY=
+
+# For debugging
+VITE_DEBUG=true
+
+# @NOTE Only on production
+# It became apparent that we need a "live" way to check what commit we're running on while debugging issues strait into production
+VITE_PROD_COMMIT_HASH=
```

Comment:
This will need to be injected automatically somehow. Either by vite or by Docker, but it's really not an option to add this manually each time

Reply:
My intention was for this file to serve as an example for all the environment variables that are available. It is indeed being set in the Dockerfile.
If you want, I can remove it here.

Reply:
Fine then. As long as I't automatic I have no issues.